package.json

```diff
@@ -75,6 +75,7 @@
     "select2": "4.0.13",
     "selectize-plugin-clear": "0.0.3",
     "sprintf-js": "1.1.2",
+    "styled-components": "6.0.8",
```

Comment:
Do try adding styled-components as a dev-dependency?

Reply:
```
"dependencies": {
  // other dependencies
},
"devDependencies": {
  "styled-components": "^6.0.8"  // ^ means any version compatible with 6.0.8
}
```


Save the package.json file after making these changes.
Open your command-line interface (CLI).
Navigate to your project's root directory.
Run the following command to remove the existing styled-components dependency:

```
npm uninstall styled-components
yarn remove styled-components
npm install styled-components --save-dev
yarn add styled-components --dev
```

Reply:
Yes, please try. But don't use yarn and don't use ^.